src/main/java/io/microprofile/config/spi/ConfigConverterHelper.java

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ * Copyright 2016 Microprofile.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ *******************************************************************************/
+
+package io.microprofile.config.spi;
+
+import java.lang.reflect.Type;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+/**
+ * The helper class returns all the built-in converters.
+ * @author <a href="mailto:emijiang@uk.ibm.com">Emily Jiang</a>
+ *
+ */
+public class ConfigConverterHelper {
+
+	static Map<Type, Converter<?>> converters = null;
+	
+	public  static  Map<Type, Converter<?>> getDefaultConverters() {
+		if (converters == null) {
```

Comment:
This is not threadsafe. I think this needs to go into a static initializer block
Also, I think it has to be an unmodifiableMap.